components/list/test/list.visual-diff.js

```diff
@@ -62,9 +62,10 @@ describe('d2l-list', () => {
 		}, y);
 	};
 
-	const wait = (milliseconds) => {
-		return page.evaluate((milliseconds) => {
-			return new Promise(resolve => setTimeout(resolve, milliseconds));
+	const wait = (selector, milliseconds) => {
+		return page.$eval(selector, async(elem, milliseconds) => {
+			await elem.updateComplete;
+			await new Promise(resolve => setTimeout(resolve, milliseconds));
```

Comment:
Is it possible the `elem.updateComplete` would be enough, and waiting 100ms isn't necessary anymore?

Reply:
It is definitely possible, but I just can't get it to reproduce consistently enough to check 😞

Reply:
Gonna go with this for now to give us our best shot at no flake.  Note to self - if we keep seeing this flake, I think the next step is to dive into the mechanics of the subscription model to see if there's any chance of a rare race condition where a subscription is simply missed, and no amount of waiting would help us.